docs/federation-changes.md

```diff
@@ -0,0 +1,174 @@
+
+### Sending an Operation to specific stores
+
+As mentioned, the `gaffer.federatedstore.operation.graphIds` option is still being temporarily supported so if you have an Operation using that option, it should continue to work. It will still work if the option is being used **inside** an OperationChain. However, if the option is being used **on** an OperationChain, then see [below](#breaking-change).  
+Despite the option being still supported, we still recommend you migrate to using a FederatedOperation.  
+
+#### Deprecated graphIds option on a single Operation
+```json
+{
+    "class": "uk.gov.gchq.gaffer.operation.impl.get.GetAllElements",
+    "options": {
+        "gaffer.federatedstore.operation.graphIds": "graphA"
+    }
+}
+```
+
+#### New FederatedOperation graphIds on a single Operation
+``` json
+{
+    "class": "uk.gov.gchq.gaffer.federatedstore.operation.FederatedOperation",
+    "operation": {
+        "class": "uk.gov.gchq.gaffer.operation.impl.get.GetAllElements"
+    },
+    "graphIds": [ "graphA" ]
+}
+```
+
+#### Deprecated graphIds option inside an OperationChain
+```json
+{
+    "class": "uk.gov.gchq.gaffer.operation.OperationChain",
+    "operations": [
+        {
+            "class": "ExampleOperation1",
+            "options": {
+                "gaffer.federatedstore.operation.graphIds": "graphA"
+            }
+        },
+        {
+            "class": "ExampleOperation2",
+            "options": {
+                "gaffer.federatedstore.operation.graphIds": "graphB"
+            }
+        }
+    ]
+}
+```
+
+#### New FederatedOperation graphIds inside an OperationChain
+```json
+{
+    "class": "uk.gov.gchq.gaffer.operation.OperationChain",
+    "operations": [
+        {
+            "class": "uk.gov.gchq.gaffer.federatedstore.operation.FederatedOperation",
+            "operation": {
+                "class": "ExampleOperation1"
+            },
+            "graphIds": [ "graphA" ]
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.federatedstore.operation.FederatedOperation",
+            "operation": {
+                "class": "ExampleOperation2"
+            },
+            "graphIds": [ "graphB" ]
+        }
+    ]
+}
+```
+
+### Breaking change
+
+#### No longer supported graphIds option on an OperationChain
+Previously, if you wanted to send an entire OperationChain to a specific graph, you could use the `graphIds` option on the chain, like so:  
+```json
+{
+    "class": "uk.gov.gchq.gaffer.operation.OperationChain",
+    "operations": [
+        {
+            "class": "uk.gov.gchq.gaffer.operation.impl.get.GetAllElements"
+        }
+    ],
+    "options": {
+        "gaffer.federatedstore.operation.graphIds": "graphA"
+    }
+}
+```
+
+#### New FederatedOperation graphIds on an OperationChain
```

Comment:
I'm not sure the this makes sense, but i could be misunderstanding.

> #### New FederatedOperation graphIds on an OperationChain
> This is no longer supported, and you should instead wrap the chain in a FederatedOperation

Makes it sound like 'New FederatedOperation graphIds on an OperationChain' isn't supported, but i'm assuming that it is supported and this is the new way of doing it.